Sources/ComposableArchitecture/Observation/ViewAction.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+/// Defines the actions that can be sent from a view.
+///
+/// See the ``ViewAction(for:)`` macro for more information on how to use this.
+public protocol ViewAction<ViewAction> {
+  associatedtype ViewAction
+  static func view(_ action: ViewAction) -> Self
+}
+
+public protocol ViewActionable<State, Action> {
```

Comment:
What do you think of `ViewActionSending` instead, since it describes that the view can send view actions?